packages/create-schemas/tests/plugins.test.ts

```diff
@@ -69,4 +70,33 @@ describe.concurrent("plugins", () => {
           "
         `);
     });
+
+    test("openapiMSWPlugin", async ({ expect }) => {
+        const plugin = experimental_openapiMSWPlugin();
+
+        assert(plugin.transform);
+
+        let emittedFile: { filename: string; code: string } | undefined;
+        function emitFile(file: { filename: string; code: string }) {
+            emittedFile = file;
+        }
+    
+        await plugin.transform({
+            config: await resolveConfig({ input: "openapi.json" }),
+            id: openapiTypeScriptId,
+            code: "export interface paths {}",
+            filename: openapiTypeScriptFilename,
+            emitFile
+        });
+
+        assert(emittedFile);
+
+        expect(emittedFile.filename).toBe("openapi-msw.ts");
+        expect(emittedFile.code).toMatchInlineSnapshot(`
+          "import type { paths } from "./types.ts";
+          import { createOpenApiHttp } from "openapi-msw";
+
+          export const http = createOpenApiHttp<paths>();"
+        `);
+    });
```

Comment:
[Suggestion] : Good to have test for the main edge case: 
`should not call transform function when id does not match openapiTypeScriptId`